vllm/v1/metrics/loggers.py

```diff
@@ -330,17 +331,12 @@ def __init__(self, vllm_config: VllmConfig, engine_index: int = 0):
                 documentation="Number of accepted tokens.",
                 labelnames=labelnames).labels(*labelvalues)
 
-        #
-        # Cache config info metric
-        #
-        self.log_metrics_info("cache_config", vllm_config.cache_config)
-
     def log_metrics_info(self, type: str, config_obj: SupportsMetricsInfo):
         metrics_info = config_obj.metrics_info()
 
         name, documentation = None, None
         if type == "cache_config":
-            name = "vllm:cache_config_info"
+            name = "vllm:cache_config_info_" + str(self.engine_index)
```

Comment:
Great catch! But the metric name shouldn't include the engine index, that should be in a label instead

Looks like we're missing the standard labels on this metric:

```
        labelnames = ["model_name", "engine"]
	labelvalues = [
            vllm_config.model_config.served_model_name,
            str(engine_index)
	]
```